apps/server/src/modules/learnroom/service/coursegroup.service.ts

```diff
@@ -14,11 +17,13 @@ export class CourseGroupService {
 	}
 
 	public async deleteUserDataFromCourseGroup(userId: EntityId): Promise<number> {
+		this.logger.log({ action: 'Deleting data from CourseGroup for user ', userId });
 		const [courseGroups, count] = await this.repo.findByUserId(userId);
 
 		courseGroups.forEach((courseGroup) => courseGroup.removeStudent(userId));
 
 		await this.repo.save(courseGroups);
+		this.logger.log({ action: 'Deleted data from CourseGroup for user ', userId });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).

Reply:
OK I can only now add info about number of updated courseGroup without userId. More deteiled information about updated documents will be added in ticket https://ticketsystem.dbildungscloud.de/browse/BC-6114